tests/test_cli.py

```diff
@@ -190,10 +193,66 @@ def test_invalid_lang(self, sample_pdf: str) -> None:
         result = self.run_cli([sample_pdf, "--ocr-lang", "piglatin"])
         result.assert_failure()
 
+    @pytest.mark.reference_generator
```

Comment:
Note, we should register this mark, else `pytest` throws this warning:

```
tests/test_cli.py:196
  /home/user/dangerzone/tests/test_cli.py:196: PytestUnknownMarkWarning: Unknown pytest.mark.reference_generator - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.reference_generator
```

See also: https://docs.pytest.org/en/stable/how-to/mark.html#registering-marks

Reply:
Thanks, check out 68505bd625859ecdef7a06ac1734728f4e58ba79

Reply:
I still see this warning here: https://github.com/freedomofpress/dangerzone/actions/runs/13990359427/job/39172782169?pr=1089#step:10:940

I think you need to switch `-` with `_` in the mark.